test/unit/enrichment/create-invoice-number.test.js

```diff
@@ -61,6 +63,11 @@ describe('create invoice number', () => {
     expect(result).toEqual('F000000100000001V001')
   })
 
+  test('generate invoice number for Manual Invoice', () => {
+    const result = createInvoiceNumber(manualPaymentRequest)
+    expect(result).toEqual('S1234567S1234567V001')
```

Comment:
Could use the mock here as we're not expecting any change.  

eg

`expect(result).toEqual(manualPaymentRequest.invoiceNumber)`